model/src/guild/permissions.rs

```diff
@@ -40,6 +40,8 @@ bitflags! {
         const MANAGE_EMOJIS = 0x4000_0000;
         const USE_SLASH_COMMANDS = 0x8000_0000;
         const REQUEST_TO_SPEAK = 0x10000_0000;
+        /// Allows for deleting and archiving threads, and viewing all private threads.
```

Comment:
Maybe something along the lines of "Allows the deletion and archiving of threads as well as the viewing of private threads."?

Reply:
I copied that from the linked issue, and i think it's clear enough